### PR TITLE
First draft.

### PR DESCRIPTION
**Pull request by jonathanpallant**
Mostly a skeleton, with how-can-i and is-the-juice mostly written.

Comes out as a book and a slide deck.

### CODE REVIEW

File: src/whos-in-charge-now.md
Comment (miguelraz) [inline src/whos-in-charge-now.md:92]: We had a really nice chat about the ecosystem, meetups, and the code of conduct in our last training.
I would at least add something about the Code of Conduct, and have some presenter's notes on the meetups/ecosystem if people ask something in that direction.

File: src/what-does-rust-run-on.md
Comment (miguelraz) [inline src/what-does-rust-run-on.md:16]: I would make an emphasis (or speaker's note) here on the fact that Rust can enter any part of your stack and help you build fast, safe, reliable tooling and applications for all production OS's, so it's a great efficiency multiplier across an organization when cross-platform concerns are minimized.

File: src/whos-in-charge-now.md
Comment (jonathanpallant) [reply src/whos-in-charge-now.md:92]: A tricky topic, but I've tried to add some words on this.

Comment (miguelraz) [reply src/whos-in-charge-now.md:92]: Straight and to the point. Great!

File: src/whats-the-catch.md
Comment (miguelraz) [inline src/whats-the-catch.md]: classes,*  templates

**Comment by miguelraz** [comment]
I was itching for something to be added and then I go to the "Is the juice worth the squeeze section" and it is amazing.

There's 1 speaker's note/chart that I would think about adding which is taking the 70% of all bugs at FAANG being memory safety and then making the argument of 
```
Rust can save the best paid programmers in the world from shipping ~70% of their bugs being memory safety. 
That is a lot of decades of expertise  of C/C++ that you can save on immediately by using Rust!
```

I'm pretty sure it would come out in some fashion, but if I was presenting I would want a visual aid to make the point for me. You choose what suit you.

To your previous comment - I think this mdbook is perfect for giving you talking-time leeway. I can see myself presenting comfortably on the pre/post code bits for an hour, or adapting on the fly depending on the audience.

I have no further comments, LGTM like this.

File: README.md
Comment (Mirabellensaft) [inline README.md]:
```suggestion
Rust for your next project. The agenda includes two interactive lectures, an extended live
```

File: README.md
Comment (Mirabellensaft) [inline README.md:53]: The subtitles use 'it' a lot to refer to Rust. It would probably be better to change it to "Rust" to make it explicit what is meant, and you also want to link the positive properties (fast, safe etc) to the name, even if you're posing a question.

File: src/is-the-juice-worth-the-squeeze.md
Comment (Mirabellensaft) [inline src/is-the-juice-worth-the-squeeze.md]: Should the . be a comma?

File: src/is-the-juice-worth-the-squeeze.md
Comment (Mirabellensaft) [inline src/is-the-juice-worth-the-squeeze.md:67]: The many brackets make this hard to read, but this is a quote, right?

File: src/what-about-safety.md
Comment (Mirabellensaft) [inline src/what-about-safety.md]: I was in support of renaming the mutable/immutable variables for the stated reasons, but here without much context I found the renaming very confusing, because `exclusive` and `shared` says nothing about mutability, and it's easy to read the exact opposite into them: shared is mutable by many, exclusive is not mutable. And I think the mutability part is still an important distinction. This is probably worth some more internal discussion.

File: src/whats-the-catch.md
Comment (Mirabellensaft) [inline src/whats-the-catch.md]: nor?
```suggestion
* Neither Rust nor C++ have a stable ABI
```

File: src/where-did-it-come-from.md
Comment (Mirabellensaft) [inline src/where-did-it-come-from.md]: Redundant information

File: src/where-did-it-come-from.md
Comment (Mirabellensaft) [inline src/where-did-it-come-from.md]:
```suggestion
  * garbage collector
  * evented runtime
  * complex error handling
```

File: src/where-did-it-come-from.md
Comment (Mirabellensaft) [inline src/where-did-it-come-from.md]: A full sentence like
```suggestion
* Rust orientated towards a usable systems programming language.
```

File: src/whats-the-catch.md
Comment (Mirabellensaft) [inline src/whats-the-catch.md:15]: ... and prioritizes managing it. 
Maybe a conclusion for the slide notes.

**Review by Mirabellensaft — COMMENTED** [review]
I have to but more general thought into the presentation of the info on the history slides. But other than that I really like what you created here.

**Review by Mirabellensaft — APPROVED** [review]

File: src/is-the-juice-worth-the-squeeze.md
Comment (jonathanpallant) [reply src/is-the-juice-worth-the-squeeze.md:67]: Yeah :(

File: src/is-the-juice-worth-the-squeeze.md
Comment (jonathanpallant) [reply src/is-the-juice-worth-the-squeeze.md]: Transcribed accurately, but I'm happy to fix Mark's typo for the sake of understanding

File: src/what-about-safety.md
Comment (jonathanpallant) [reply src/what-about-safety.md]: Attempted to clarify this.

File: src/where-did-it-come-from.md
Comment (jonathanpallant) [reply src/where-did-it-come-from.md]: I think it's still useful to show that Rust once had "more" and people modified it to have "less", as opposed to Rust only having "less" because people haven't gotten around to adding "more".

Comment (jonathanpallant) [reply src/where-did-it-come-from.md]: Unless you mean the bullet point itself, in which case, yeah I can buy that.

File: src/whats-the-catch.md
Comment (jonathanpallant) [reply src/whats-the-catch.md:15]: How do I add slide notes?

Comment (Mirabellensaft) [reply src/whats-the-catch.md:15]: That is probably a question for @miguelraz

File: src/what-about-safety.md
Comment (Mirabellensaft) [inline src/what-about-safety.md:20]: excellent!

**Review by Mirabellensaft — APPROVED** [review]

**Comment by jonathanpallant** [comment]
@Mirabellensaft is this OK after my changes?

**Comment by Mirabellensaft** [comment]
I am in the middle of taking a pass through the rendered book, but content wise it's fine as far as I can judge without seeing it play out.

File: src/what-about-safety.md
Comment (Mirabellensaft) [inline src/what-about-safety.md]: ?
```suggestion
* You can't 'accidentally' go off the end of a buffer
```

**Comment by Mirabellensaft** [comment]
I found one other thing, but other than that, this is good to go! Let me know if you want to do a dry run some time.

**Comment by jonathanpallant** [comment]
@Mirabellensaft I merged the last change. Can you click approve?

**Review by Mirabellensaft — APPROVED** [review]

**Comment by Mirabellensaft** [comment]
Hm. It seems like my review is not enough?

**Review by miguelraz — APPROVED** [review]

**Pull request merged**
